yarn.lock

```diff
@@ -5,6 +5,7 @@
 "@babel/cli@^7.1.0":
   version "7.1.2"
   resolved "https://botbuilder.myget.org/F/botbuilder-v4-js-daily/npm/@babel/cli/-/@babel/cli-7.1.2.tgz#fc2853ae96824b3779ca85de4fd025ce3cf62a5e"
+  integrity sha1-/ChTrpaCSzd5yoXeT9Alzjz2Kl4=
```

Comment:
@wsguede something you have is adding these integrity checks.  I'm worried heroku may fail since I can't/haven't set the npm config here.

Reply:
FYI I'm yarn 1.5.1.  We can always try and see what heroku says